CHANGELOG.md

```diff
@@ -1 +1,107 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
```

Comment:
Let's finally lose the "please" from here and from the template you're using.

Also change the last part to "where you can find target removal dates."